project/plugins.sbt

```diff
@@ -6,3 +6,4 @@ addSbtPlugin("com.earldouglas"      % "xsbt-web-plugin"   % "2.1.1")
 addSbtPlugin("fi.gekkio.sbtplugins" % "sbt-jrebel-plugin" % "0.10.0")
 addSbtPlugin("com.typesafe.sbt"     % "sbt-pgp"           % "0.8.3")
 addSbtPlugin("io.get-coursier"      % "sbt-coursier"      % "1.0.0-M15")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
```

Comment:
I prefer that helper plugin like this is defined in user local sbt settings (`~/.sbt/0.13/plugins/build.sbt `).